cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -325,12 +368,89 @@ def import_status_handler(request, course_key_string, filename=None):
     return JsonResponse({"ImportStatus": status})
 
 
+def library_import_status_handler(request, library_key_string, filename=None):
+    """
+    Shim to make the status handler work with libraries.
+    """
+    return import_status_handler(request, library_key_string, filename)
+
+
+def create_export_tarball(course_module, course_key, context, library=False):
+    """
+    Generates the export tarball, or returns None if there was an error.
+
+    Updates the context with any error information if applicable.
+    """
+    name = course_module.url_name
+    export_file = NamedTemporaryFile(prefix=name + '.', suffix=".tar.gz")
+    root_dir = path(mkdtemp())
+
+    try:
+        if library:
```

Comment:
Change this to `if isinstance(course_key, LibraryLocator)` and you could get rid of the library argument.